tests/devices/test_default_mixed_autograd.py

```diff
@@ -120,7 +120,7 @@ class TestDtypePreserved:
     """Test that the user-defined dtype of the device is preserved for QNode
     evaluation"""
 
-    @pytest.mark.parametrize("r_dtype", [np.float32, np.float64])
+    @pytest.mark.parametrize("r_dtype_name", ["float32", "float64"])
```

Comment:
Why this change?

Reply:
I actually don't know. I think some test was failing and I thought this fixed it. Probably just a red herring. I reverted the change